content/en/agent/amazon_ecs/data_collected.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Amazon ECS Data Collection
+kind: documentation
+further_reading:
+- link: "/agent/amazon_ecs/logs/"
+  tag: "Documentation"
+  text: "Collect your application logs"
+- link: "/agent/amazon_ecs/apm/"
+  tag: "Documentation"
+  text: "Collect your application traces"
+- link: "/agent/amazon_ecs/metrics/"
+  tag: "Documentation"
+  text: "Collect ECS metrics"
+---
+
+## Data collected
+
+### Metrics
+
+See [metadata.csv][1] for a list of metrics provided by this integration.
+
+Each of the metrics retrieved from AWS is assigned the same tags that appear in the AWS console, including but not limited to hostname, security-groups, and more.
+
+**Note**: Metrics prefixed with `ecs.containerinsights.*` come from the [AWS CloudWatch agent][2].
+
+### Events
+
+To reduce noise, the Amazon ECS integration is automatically set up to include only events that contain the following words: `drain`, `error`, `fail`, `insufficient memory`, `pending`, `reboot`, `terminate`. See example events below:
```

Comment:
I believe that this is web integration too

Reply:
Is there someone we can verify this with? It says `Amazon ECS integration` in the pretext, so I want to verify if we should keep / remove.

Reply:
That would be @jimmycaputo the PM for AWS integrations. We should make sure that the anything we remove from the Agent doc for ECS is still available on the integration page for the web integration

Reply:
I'm not sure I understand the question. Can one of you clarify?

Reply:
@jimmycaputo - we're looking to clarify whether this is specific to the AWS Web integration, and whether it should be documented within the Amazon ECS docs.

Reply:
Likewise with the above 2 comments. It looks like the metadata.csv is pulling in the AWS Integration metrics.